dev/source/docs/static-analysis.rst

```diff
@@ -20,7 +20,7 @@ You can do your own static analysis using autotest:
 
    You may need to install clang for this to work.
 
-   On Ubuntu, "sudo apt-get install clang"
+   On Ubuntu, `sudo apt-get install clang-7 clang-tools-7`
```

Comment:
Is this going to be wrong on older versions of Ubuntu?

Reply:
I have no idea, i'm on 18.04 and it worked for me. We could just mention the need for it and not provide the command.....

Reply:
yep, doesn't work on 20.04 ...
so `sudo apt-get install clang clang-tools` will get you the latest available on ubuntu

Reply:
removed the version numbers, should work for everyone ?